### ISSUE TITLE
TypeError: e.services.map is not a function

### ISSUE THREAD

**Issue by binduwavell** [getferdi/ferdi#1773]
I exported my ferdi config from `http://localhost:45569/transfer` and sent it to a new computer. I then attempt to import the file from the same link on a new computer and I get this error:

```
Could not import your workspaces into our system.
Error: TypeError: e.services.map is not a function
```

The source machine is on Windows while the target machine is a Mac. I believe both have the latest versions of Ferdi installed. Any advice on how to debug this issue would be greatly appreciated!

**Comment by binduwavell** [comment]
Oh so fun times, I import twice with that error. I reloaded Ferdi and now I have all my services twice and no workspaces. I can work with this. But it would be good if the import were idempotent and if workspaces worked :)

**Comment by vraravam** [comment]
thanks for reporting this @binduwavell .

The idempotency is not possible - since the import feature is **supposed to be additive** for other services that the user might have setup manually.

For the issue with the exception, I will try to see if I can solve it. Would you mind sharing the exported data file? (It doesn't have anything sensitive - feel free to edit the parts that might have your PII data)

**Comment by vraravam** [comment]
Also - could you please specify which version of Ferdi you are on?

**Comment by vraravam** [comment]
This fix will be in the next nightly: `v5.6.1-nightly.24`
